import-export-cli/mi/impl/common.go

```diff
@@ -165,6 +165,23 @@ func invokeDELETERequestWithRetry(url string, env string) (*resty.Response, erro
 	})
 }
 
+func invokeDELETERequestWithRetryAndParams(url string, env string, params map[string]string) (*resty.Response, error) {
```

Comment:
```suggestion
func invokeDELETERequestWithRetryAndParams(url, env string, params map[string]string) (*resty.Response, error) {
```
Shall we refactor the method signature by grouping the string arguments together, as shown above, to reduce the number of characters per line?

Reply:
fixed in 84dfebb1a9d2419c951bfbb295afe6ccd5c8dfe2